Geometry/HGCalCommonData/src/HGCalParametersFromDD.cc

```diff
@@ -52,11 +59,34 @@ bool HGCalParametersFromDD::build(const DDCompactView* cpv,
   DDSpecificsMatchesValueFilter filter{val};
   DDFilteredView fv(*cpv,filter);
   bool ok = fv.firstChild();
+  int  mode(0);
 
   if (ok) {
     DDsvalues_type sv(fv.mergedSpecifics());
-    php.mode_ = getGeometryMode("GeometryMode", sv);
+    php.mode_ = getGeometryMode("GeometryMode", sv, true);
```

Comment:
Hi @ianna - right, thats what I'm afraid of - are we mixing an int (now an enum) and a bool type? (answer: yes) Is that required for some reason?